### PR TITLE
WRRZ new station with broadband and SM

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Opening new Station WRRZ `White Rock Road`
 - Opening new Broad band site + stream on LC 10:
   - Sensor: Trillium Horizon TH120-2 (3354)
   - Datalogger: Centaur CTR4-6S (10959), firmware 4.8.5
- Opening new strong motion site + stream on LC 20:
  - Recorder: TitanSMA (002215), firmware 4.5.22

### CODE REVIEW

**Review by mnaguit — COMMENTED** [review]
Thanks @Thomas-Benson, can you please double check the following:
- do we need to add role in the dataloggers as broadband? probably not, just checking
- for the station coordinate, the longitude is incorrect (i know, @gva6500 has same lat/long in the main ticket 😁 )

Other than those, I think all good! great job! 👍

File: network/stations.csv
Comment (salichon) [inline network/stations.csv:1]: oops :) Longitude aye

**Review by salichon — DISMISSED** [review]
Hello @Thomas-Benson 
i reckon the station coordinates had a hickhock :) 
The station might be endeing on the other side of the world :))

**Comment by Thomas-Benson** [comment]
Coordinates updated. I caught it for the sites.csv, but forgot to change it for the station entry 🤦 
And @mnaguit the role shouldn't be neccessary unless there is more than one datalogger at the site.

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]

File: network/stations.csv
Comment (Thomas-Benson) [reply network/stations.csv:1]: Longitude copy-past typo fixed

**Pull request merged**